.prettierignore

```diff
@@ -3,3 +3,4 @@
 client/static
 fixtures
 curriculum/challenges/_meta/*/*
+curriculum/challenges/*
```

Comment:
```suggestion
curriculum/challenges/*
```
So, this works, but according to my reading of the `.gitignore` docs, this should not match anything deeper than `curriculum/challenges/english`.  What am I missing?

Reply:
Magic, apparently - because yes this should have been `curriculum/challenges/**/*`